TypeInjections/Tool/Tool.fs

```diff
@@ -4,39 +4,88 @@
 
 namespace Tool
 
+open LibGit2Sharp
 open System.IO
 open System
 open TypeInjections
 
 module Tool =
     
+    //NOTE 'location' refers to either the github url OR path to repo on local computer
+    let getCommit (SHA: string) (location:string) (outputPath:string) : unit =
+      Utils.log ("path to repo is " + outputPath)
```

Comment:
this can be `Utils.log $"path to repo is {outputPath}"`

Reply:
Fixed